arche/test/troy_west/arche_integration_test.clj

```diff
@@ -33,22 +33,26 @@
   (ccm/stop!)
   (reset! system {}))
 
+(use-fixtures :each (fn [test-fn]
+                      (when-not @system
+                        (start-system!)
+                        (test-fn))))
```

Comment:
i'm only making this comment because Tristan told me the test-fn should be bumped one out.